ownership-chain/runtime/laos/src/configs/fee/mock.rs

```diff
@@ -0,0 +1,107 @@
+use frame_support::{
```

Comment:
I also don't think it's necessary to have a mock runtime for this. imagine we want to write similar tests for other pallet or type:
1. we create another mock runtime to it
2. realize we have too many mock runtimes and try unify both mock runtime
3. continue like this for several pallets and we end up with a mock runtime which is pretty much the same as the actual runtime

So, I suggest to simply use the actual runtime with `new_test_ext()`